swagger.yaml

```diff
@@ -581,6 +581,87 @@ paths:
           required: true
           schema:
             type: string
+      responses:
+        201:
+          description: Success.
+        400:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        401:
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/infra_error'
+        403:
+          description: Forbidden.
+          schema:
+            $ref: '#/definitions/infra_error'
+        404:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        405:
+          description: Method Not Allowed.
+          schema:
+            $ref: '#/definitions/error'
+        500:
```

Comment:
please add 503 as well to both apis

Reply:
I thought those were only needed for endpoints with `agentAuth`?

Reply:
Right